tests/test_decorator.py

```diff
@@ -417,7 +376,7 @@ def func(s: str, count: int, *, separator: bytes = b''):
     stub.assert_not_called()
 
 
-@pytest.mark.xfail(reason='working on V2')
+@pytest.mark.xfail(reason='Annotated does not seem to be respected')
```

Comment:
Is it the `@validate_arguments` decorator that is not respecting `Annotated`? Or are our models not respecting it? Asking because `@validate_arguments` builds a model internally.

Reply:
Annotated definitely works on BaseModel fields, and I checked it seems to work for this sort of case:
```python
from typing import Annotated

from pydantic import BaseModel, Field
class Model(BaseModel):
    x: Annotated[int, Field(alias='b')]

print(Model.model_fields)
# {'x': FieldInfo(annotation=int, required=True, alias='b', alias_priority=2)}
```

Reply:
We honestly should probably just rework `@validate_arguments` internally to create a `Validator` for each argument instead of a model, or something like that.